### ISSUE TITLE
add support for node_entity_queue

### ISSUE THREAD

**Issue by Niklp09** [mt-mods/signs_lib#13]
https://github.com/fluxionary/minetest-node_entity_queue
This may increase performance if there are many signs that need to be loaded at the same time.

@fluxionary ping

**Comment by fluxionary** [comment]
yeah, i was going to get around to adding support here eventually, but i wanted to develop the API a bit more. i feel like it's still a bit crude.

**Comment by wsor4035** [comment]
looking over at that, what it could do to help signs lib out is very much and edge case. also probably would be better to implement the rough concept rather than dep on it

**Comment by fluxionary** [comment]
> looking over at that, what it could do to help signs lib out is very much and edge case. also probably would be better to implement the rough concept rather than dep on it

it can absolutely be done as an optional dep, this is how i currently do my smartshop & cottages work
